evals/runner.py

```
@@ -0,0 +1,392 @@
+        print(
+            f"{r.id.ljust(width_id)}  {r.category.ljust(width_cat)}  "
+            f"{mark}    {time_s}  {tokens}  {r.detail[:80]}"
+        )
+    print("-" * 90)
+    print(f"\n{pass_count}/{len(results)} passed")
+
+
+def _save_report(results: list[CaseResult], path: Path) -> None:
+    path.parent.mkdir(parents=True, exist_ok=True)
+    payload = {
+        "ts": datetime.now(timezone.utc).isoformat(),
```

Ruff (0.15.12)

[warning] Use `datetime.UTC` alias

Convert to `datetime.UTC` alias

(UP017)